### PR TITLE
npm update

### PR DESCRIPTION
**Pull request by brandones**

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/spa/SpaActivator.java [hunk 1]
```diff
@@ -11,12 +11,9 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.openmrs.api.AdministrationService;
-import org.openmrs.api.context.Context;
 import org.openmrs.module.BaseModuleActivator;
 import org.openmrs.module.ModuleException;
 import org.openmrs.module.spa.utils.SpaModuleUtils;
-import org.openmrs.util.OpenmrsUtil;
```
Comment (brandones) [inline omod/src/main/java/org/openmrs/module/spa/SpaActivator.java:19]: unrelated cleanup

File: omod/src/main/webapp/resources/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "spa-core-resources",
   "license": "MPL-2.0",
   "dependencies": {
-    "import-map-overrides": "1.14.2",
-    "systemjs": "6.1.7"
+    "import-map-overrides": "^1.14.3",
+    "systemjs": "^6.3.0"
```
Comment (brandones) [inline omod/src/main/webapp/resources/package.json:6]: this is the substantive change

**Review by mogoodrich — COMMENTED** [review]
Did you mean to commit the node_modules folders for import-map-override and systemjs?

**Comment by brandones** [comment]
Yep, they live statically in this module's "resources."
